File: src/API/WesternStatesWater.WestDaat.Contracts.Client/Requests/Admin/OrganizationMemberAddRequest.cs
namespace WesternStatesWater.WestDaat.Contracts.Client.Requests.Admin;

public class OrganizationMemberAddRequest : OrganizationStoreRequestBase
{
    public Guid OrganizationId { get; set; }

    public Guid UserId { get; set; }

    public string Role { get; set; }
}

File: src/DashboardUI/src/data-contracts/OrganizationMemberAddRequest.ts
import { Role } from '../config/role';
import { OrganizationStoreRequestBase } from './OrganizationStoreRequestBase';

export interface OrganizationMemberAddRequest extends OrganizationStoreRequestBase {
  $type: 'OrganizationMemberAddRequest';
  userId: string;
  role: Role;
}
